test/test_continue_tabulation_config.json

```diff
@@ -1,6 +1,4 @@
 {
-  "visualizerOutputFilename":"test_continue_tabulation_visualizer.xlsx",
-  "auditOutputFilename":"minneapolis_2013_audit_log.txt",
   "outputDirectory":"test",
```

Comment:
to what?

Reply:
I was thinking let's not put test output in the test input directory.

Reply:
Ah, I see now someone has made a test_output directory which I assume it what you're expecting.  What is your vision for the testing framework?  I was thinking something where each test case (or perhaps each set of related test cases) has its own folder, e.g. /tests/continue_tabulation/...   and all inputs, outputs, and expected results go into that folder.  This would facilitate comparing outputs to expected outputs, until robots replace us all.

Reply:
I'm fine with whatever as long as it's easy to keep the output out of the repo using .gitignore. I just don't think it should go directly in the test directory.

Reply:
sounds like ur not fine with whatever...